### PR TITLE
:bug: PRTL-1176 keep searchTableTab after CurrentFilters clear

### PR DESCRIPTION
**Pull request by cy**

### CODE REVIEW

File: modules/node_modules/@ncigdc/components/CurrentFilters.js [hunk 1]
```diff
@@ -130,7 +132,7 @@ const CurrentFilters = ({
         <Row wrap spacing="0.3rem">
           <NotUnderlinedLink
             style={styles.groupPadding}
-            query={{}}
+            query={{ searchTableTab }}
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/CurrentFilters.js]: It should probably keep other things like the facet tab as well.
Might be better to specify the things we are clearing instead? either by passing in the whole query and omitting filters or making `query={{filters: null}}` work.

**Comment by cy** [comment]
@Jephuff ready for re-review

File: modules/node_modules/@ncigdc/components/CurrentFilters.js [hunk 2]
```diff
@@ -57,6 +58,7 @@ const LinkButton = styled(Link, {
 });
 
 type TProps = {
+  searchTableTab: string,
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/CurrentFilters.js]: this isn't used any more is it?

Comment (cy) [inline modules/node_modules/@ncigdc/components/CurrentFilters.js]: opps, updated

**Pull request merged**